Resources/Prototypes/_ES/Entities/Structures/Power/antimatter.yml

```diff
@@ -0,0 +1,193 @@
+    fixtures:
+      fix1:
+        hard: false
+        density: 7
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.45,-0.45,0.45,0.45"
+        layer:
+        - MidImpassable
+        - Impassable
+  - type: AtmosDevice
+  - type: EdgeSpreader
+    id: ESAntimatter
+  - type: EventHorizon
+    consumerFixtureId: fix1
+    consumeTiles: false
+  - type: Explosive
+    explosionType: Default
+    totalIntensity: 60
+    intensitySlope: 5
+    maxIntensity: 30
+    canCreateVacuum: false
+
+- type: entity
+  id: ESNegentropyAccumulator
+  name: negentropy accumulator
+  description: A device capable of neutralizing antimatter to produce large bursts of energy.
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    sprite: _ES/Structures/Power/negentropy.rsi
+    noRot: true
+    layers:
+    - state: icon
+      map: ["base"]
+    - state: draining
+      visible: false
+      shader: unshaded
+      map: ["glow"]
+  - type: Appearance
+  - type: GenericVisualizer
+    visuals:
+      enum.ESAntimatterConverterVisuals.Draining:
+        glow:
+          True: { visible: true }
+          False: { visible: false }
+      enum.ESAntimatterConverterVisuals.Broken:
+        base:
+          True: { state: broken }
+          False: { state: icon }
+  - type: ESAntimatterConverter
+  - type: NodeContainer
+    examinable: true
+    nodes:
+      input:
+        !type:CableDeviceNode
+        nodeGroupID: HVPower
+  - type: PowerMonitoringDevice
+    group: Generator
+    loadNode: input
+    collectionName: negentropy
+    sprite: _ES/Structures/Power/negentropy.rsi
+    state: draining
+  - type: Battery
+    maxCharge: 5000000
+    startingCharge: 0
+  - type: BatteryDischarger
+  - type: PowerNetworkBattery
+    maxSupply: 350000
+    supplyRampTolerance: 350000
+  - type: PointLight
+    enabled: false
+    radius: 1.5
+    energy: 3
+    color: "#00AAFF"
+  - type: Transform
+    anchored: true
+  - type: Physics
+    bodyType: Static
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.30
+        density: 110
+        mask:
+        - MachineMask
+        layer:
+        - MachineLayer
+  - type: Anchorable
+  - type: Rotatable
+  - type: Pullable
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Repairable
+    fuelCost: 15
+    doAfterDelay: 3
+  - type: Damageable
+    damageContainer: StructuralInorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 50
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          collection: MetalBreak
+      - !type:DoActsBehavior
+        acts: ["Breakage"]
+
+- type: entity
+  parent: BaseStructure
+  id: ESAntimatterCapsule
+  name: antimatter capsule
+  description: A highly-experimental container used for transporting antimatter. Safe enough, short of throwing explosives at it. But you'd never do that...
```

Comment:
seems weird for this to be the description when the intent is that you -should- blow it up

Reply:
overall this fact doesnt seem communicated very well (i know it doesnt have guidebooks yet but)